packages/sdk-client-v3/src/types/types.d.ts

```diff
@@ -236,11 +236,9 @@ export type HttpOptions = {
 export type LogLevel = 'INFO' | 'ERROR'
 
 export type LoggerMiddlewareOptions = {
-  logLevel?: LogLevel
   maskSensitiveHeaderData?: boolean
   includeOriginalRequest?: boolean
   includeResponseHeaders?: boolean
-  includeRequestInErrorResponse?: boolean
```

Comment:
Why was this remove, I am curious.

Reply:
Because it was not used.

Reply:
Okay, I think we need a comment or at least a ticket for this feature.

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/708/commits/ef33f292bcd1d7f8f6440189fb480a9f79b05d75